config/config.exs

```diff
@@ -44,7 +44,7 @@ end
 config :mbta_metro, :map, %{
   center: [-71.0589, 42.3601],
   maxZoom: 18,
-  minZoom: 12,
+  minZoom: 9,
```

Comment:
```suggestion
  minZoom: 8,
```

I feel like we could go further here, thinking about even longer trips on the smallest screen. Though I'm not sure it's even worth enforcing a minZoom at all...

Reply:
Yeah good point. Logan to Providence still doesn't fit with a minZoom of 9.

I agree about not enforcing a minZoom. I wonder what happens if we just drop that param...

Reply:
<img width="566" alt="Screenshot 2025-01-09 at 2 36 15 PM" src="https://github.com/user-attachments/assets/318e8257-eca8-4022-814f-1c39a70562d3" />

Eeeeeehhhhhh, this isn't the best thing in the world.

Reply:
<img width="1003" alt="Screenshot 2025-01-09 at 2 38 10 PM" src="https://github.com/user-attachments/assets/f6751d99-19e0-4c51-9f21-d36dbe99b22d" />

Zoom level 7

<img width="986" alt="Screenshot 2025-01-09 at 2 39 29 PM" src="https://github.com/user-attachments/assets/19925897-206a-44df-ba46-bd102f8276f5" />